Formula/x3270.rb

```diff
@@ -10,17 +10,18 @@ class X3270 < Formula
     sha256 "897d18de2b85172e751719687710f20b1717293b10dac669d6eb052e08d4b02a" => :yosemite
   end
 
+  option "without-x11", "Exclude x3270 (X11-based version)"
```

Comment:
> Also, the name should probably be with-x3270 then, and if so, how do I bind a dependency to an option?

You can't, unfortunately. This is a limitation of our dependency system.

Go for:
`option "with-x11", "Include x3270 (X11-based version)"`